### PR TITLE
Add "image.pullSecrets" to helm values to specify imagePullSecrets for deployment and hook jobs

### PR DESCRIPTION
**Pull request by dragon3**
## WHAT
This pull request adds "image.pullSecrets" to helm values to specify imagePullSecrets for deployment and hook jobs.

## WHY
Since we might use in-house registry for docker images, we need to specify the imagePullSecrets to get those images.

### CODE REVIEW

File: kritis-charts/templates/kritis-server-deployment.yaml [hunk 1]
```diff
@@ -24,6 +24,12 @@ spec:
         label: {{ .Values.serviceLabel }}
         release: {{ .Release.Name }}
     spec:
+      {{- if .Values.image.pullSecrets }}
+      imagePullSecrets:
+      {{- range .Values.image.pullSecrets }}
+      - name: {{ . }}
+      {{- end}}
```
Comment (vbanthia-zz) [inline kritis-charts/templates/kritis-server-deployment.yaml]: space after `end`

File: kritis-charts/templates/postinstall/1_job.yaml [hunk 2]
```diff
@@ -16,6 +16,12 @@ spec:
       labels:
         app: {{ .Values.postinstall.job.name }}
     spec:
+      {{- if .Values.image.pullSecrets }}
+      imagePullSecrets:
+      {{- range .Values.image.pullSecrets }}
+        - name: {{ . }}
+      {{- end}}
```
Comment (vbanthia-zz) [inline kritis-charts/templates/postinstall/1_job.yaml]: space after `end`

File: kritis-charts/templates/predelete/1_job.yaml [hunk 3]
```diff
@@ -17,6 +17,12 @@ spec:
       labels:
         app: {{ .Values.predelete.job.name }}
     spec:
+      {{- if .Values.image.pullSecrets }}
+      imagePullSecrets:
+      {{- range .Values.image.pullSecrets }}
+        - name: {{ . }}
+      {{- end}}
```
Comment (vbanthia-zz) [inline kritis-charts/templates/predelete/1_job.yaml]: space after `end`

File: kritis-charts/templates/preinstall/3_job.yaml [hunk 4]
```diff
@@ -16,6 +16,12 @@ spec:
       labels:
         app: {{ .Values.preinstall.job.name }}
     spec:
+      {{- if .Values.image.pullSecrets }}
+      imagePullSecrets:
+      {{- range .Values.image.pullSecrets }}
+        - name: {{ . }}
+      {{- end}}
```
Comment (vbanthia-zz) [inline kritis-charts/templates/preinstall/3_job.yaml]: space after `end`

**Review by vbanthia-zz — DISMISSED** [review]
look good to me with some NIT comments.

**Comment by dragon3** [comment]
@vbanthia 
Thank you for the review.  I've fixed the space in 184bff5.

**Review by vbanthia-zz — APPROVED** [review]

**Comment by dragon3** [comment]
Thanks!
I'm going to merge it.

**Pull request merged**